databricks_blueprints/helpers.py

```diff
@@ -1,5 +1,6 @@
 import sys
 import requests
+from pathlib import Path
```

Comment:
Can we rename this file to client? Generally the name helpers is too ... general. And the point of the module is to expose our DatabricksClient.